dev/vagrant/conf/nomad/README.md

```diff
@@ -1,10 +1,7 @@
-# Overriding and appending nomad configuration
-
-You may add any hcl-files to this directory in order to change the configuration.
-Any valid configuration added to this directory will append the configuration, in lexical order.
-
-Adding a file `99-override.hcl` you will ensure it will be appended last, and 00-override.hcl will be read first.
-Any valid configuration from [https://www.nomadproject.io/docs/configuration#general-parameters](https://www.nomadproject.io/docs/configuration#general-parameters) will work.
+# Nomad configuration 
+You may add any `.hcl` or `.json` -files to this directory in order to change or override the configuration.
+Any valid [nomad configuration](https://www.nomadproject.io/docs/configuration#general-parameters) added to this directory will append to the configuration, in lexical order. 
+It means that by adding the file `00-override.hcl` will be read first and the file `99-override.hcl` you will ensure it will be read last.
```

Comment:
```suggestion
It means that by prefixing a filename with `00` (e.g. `00-override.hcl`), it will be read first and using `99` (e.g. `99-override.hcl`) it will be read last.
```